backend/nodes/image_adj_nodes.py

```diff
@@ -1,219 +1,225 @@
-            img[:, :, :3] = cv2.addWeighted(
-                img[:, :, :3], alpha_c, img[:, :, :3], 0, gamma_c
-            )
-        img = np.clip(img, 0, 1).astype("float32")
-
-        return img
-
-
-@NodeFactory.register("chainner:image:threshold")
-class ThresholdNode(NodeBase):
-    """OpenCV Threshold node"""
-
-    def __init__(self):
-        """Constructor"""
-        super().__init__()
-        self.description = "Perform a threshold on an image."
-        self.inputs = [
-            ImageInput(),
-            SliderInput("Threshold", 0, 100, 50),
-            SliderInput("Maximum Value", 0, 100, 100),
-            ThresholdInput(),
-        ]
-        self.outputs = [ImageOutput()]
-        self.category = IMAGE_ADJUSTMENT
-        self.name = "Threshold"
-        self.icon = "MdShowChart"
-        self.sub = "Adjustments"
-
-    def run(
-        self, img: np.ndarray, thresh: int, maxval: int, thresh_type: int
-    ) -> np.ndarray:
-        """Takes an image and applies a threshold to it"""
-
-        logger.info(f"thresh {thresh}, maxval {maxval}, type {thresh_type}")
-
-        real_thresh = thresh / 100
-        real_maxval = maxval / 100
-
-        logger.info(f"real_thresh {real_thresh}, real_maxval {real_maxval}")
-
-        _, result = cv2.threshold(img, real_thresh, real_maxval, thresh_type)
-
-        return result
-
-
-@NodeFactory.register("chainner:image:threshold_adaptive")
-class AdaptiveThresholdNode(NodeBase):
-    """OpenCV Adaptive Threshold node"""
-
-    def __init__(self):
-        """Constructor"""
-        super().__init__()
-        self.description = "Perform an adaptive threshold on an image."
-        self.inputs = [
-            ImageInput(),
-            SliderInput("Maximum Value", 0, 100, 100),
-            AdaptiveMethodInput(),
-            AdaptiveThresholdInput(),
-            OddIntegerInput("Block Size", default=3, minimum=3),
-            IntegerInput("Mean Subtraction"),
-        ]
-        self.outputs = [ImageOutput()]
-        self.category = IMAGE_ADJUSTMENT
-        self.name = "Threshold (Adaptive)"
-        self.icon = "MdAutoGraph"
-        self.sub = "Adjustments"
-
-    def run(
-        self,
-        img: np.ndarray,
-        maxval: int,
-        adaptive_method: int,
-        thresh_type: int,
-        block_size: int,
-        c: int,
-    ) -> np.ndarray:
-        """Takes an image and applies an adaptive threshold to it"""
-
-        assert (
-            img.ndim == 2
-        ), "Image must be grayscale (single channel) to apply an adaptive threshold"
-
-        # Adaptive threshold requires uint8 input
-        img = (img * 255).astype("uint8")
-
-        real_maxval = maxval / 100 * 255
-
-        result = cv2.adaptiveThreshold(
-            img,
-            real_maxval,
-            adaptive_method,
-            thresh_type,
-            block_size,
-            c,
-        )
-
-        return result.astype("float32") / 255
+from __future__ import annotations

```

Comment:
I think you changed the line ends for this file. Please don't do that.

Reply:
I did not. This is just how git decided to treat the diff when I rebased and had to resolve the conflict with that one minor PR you made to this file.

Reply:
> This is just how git decided to treat the diff 

I am like 95% certain that you accidentally changed the line ends here. I don't know the internals of how git handles line ends in merge conflicts, but you might have to renormalize the line ends for this file.

This is not normal behavior.

> when I rebased

Rebased or just merged the main branch?

Reply:
Actually, you know what, that might have happened. I didn't like VSCode's interactive merger so I switched back to pycharm to do that specifically, but the line endings in PyCharm were never changed to LF.